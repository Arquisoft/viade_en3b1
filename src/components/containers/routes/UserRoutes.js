import React from "react";
import NavBar from '../../graphic interface/NavBar.js';
import ListUserRoutes from '../../graphic interface/ListUserRoutes.js';

const UserRoutes = () => {
  return (
    <div>
      <div>
        <NavBar />
      </div>
      <div>
        <ListUserRoutes />
      </div>
    </div>
  );
};

export default UserRoutes;