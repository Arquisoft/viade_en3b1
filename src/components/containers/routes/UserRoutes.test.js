import UserRoutes from "./UserRoutes";

test("Define UserRoutes", () => {
    expect(UserRoutes).toBeDefined();
});

test("Renders correctly", () => {
    const wrapper = render(<UserRoutes />);
    expect(wrapper).toMatchSnapshot();
});