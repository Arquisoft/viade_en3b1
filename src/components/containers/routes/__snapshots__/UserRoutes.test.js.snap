// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div>
  <div>
    <div
      class="makeStyles-root-1"
    >
      <header
        class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular makeStyles-toolbar-2 MuiToolbar-gutters"
        >
          <div
            class="MuiGrid-root makeStyles-link-4"
          >
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <a
                class="MuiTypography-root MuiLink-root MuiLink-underlineNone MuiTypography-colorInherit"
                href="#/home"
              >
                <div
                  class="MuiGrid-root MuiGrid-container"
                >
                  <div
                    class="MuiGrid-root MuiGrid-item"
                  >
                    <img
                      alt="Viade Logo"
                      src="logo_2.svg"
                      style="height:2.5rem;width:2.5rem"
                    />
                  </div>
                  <div
                    class="MuiGrid-root MuiGrid-item"
                  >
                    <h4
                      class="MuiTypography-root MuiTypography-h4"
                    >
                      ViaDe
                    </h4>
                  </div>
                </div>
              </a>
            </div>
          </div>
          <div />
        </div>
      </header>
    </div>
  </div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  >
    <div
      class="MuiGrid-root MuiGrid-item"
      hidden=""
    >
      <div>
        <main
          class="MuiGrid-root makeStyles-root-187 MuiGrid-container"
        >
          <div
            class="MuiGrid-root makeStyles-image-188 MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
          >
            <div
              class="MuiGrid-root MuiGrid-container"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
              >
                <p
                  class="MuiTypography-root makeStyles-text-189 MuiTypography-body1"
                >
                  Your routes,
                </p>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
              >
                <p
                  class="MuiTypography-root makeStyles-text2-190 MuiTypography-body1"
                >
                  your rules.
                </p>
              </div>
            </div>
          </div>
        </main>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item"
      style="max-width:75rem"
    >
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-auto"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
          >
            <div
              aria-hidden="true"
              class="MuiBackdrop-root MuiBackdrop-invisible"
              style="opacity:0;color:#5c5585"
            >
              <div
                class="MuiCircularProgress-root MuiCircularProgress-indeterminate"
                hidden=""
                role="progressbar"
                style="width:40px;height:40px"
              >
                <svg
                  class="MuiCircularProgress-svg"
                  viewBox="22 22 44 44"
                >
                  <circle
                    class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
                    cx="44"
                    cy="44"
                    fill="none"
                    r="20.2"
                    stroke-width="3.6"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-12 MuiGrid-grid-md-12"
      hidden=""
      style="margin-top:10rem"
    >
      <div>
        <footer
          class="makeStyles-footer-207"
        >
          <div
            class="MuiContainer-root MuiContainer-maxWidthSm"
          >
            <h6
              class="MuiTypography-root MuiTypography-h6"
            >
              ViaDe En3B1
            </h6>
            <h6
              class="MuiTypography-root MuiTypography-subtitle2 MuiTypography-colorTextSecondary"
            >
              Made by Pablo Ca√±al
            </h6>
            <a
              class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
              href="https://github.com/Arquisoft/viade_en3b1/"
              target="_blank"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 .3a12 12 0 0 0-3.8 23.4c.6.1.8-.3.8-.6v-2c-3.3.7-4-1.6-4-1.6-.6-1.4-1.4-1.8-1.4-1.8-1-.7.1-.7.1-.7 1.2 0 1.9 1.2 1.9 1.2 1 1.8 2.8 1.3 3.5 1 0-.8.4-1.3.7-1.6-2.7-.3-5.5-1.3-5.5-6 0-1.2.5-2.3 1.3-3.1-.2-.4-.6-1.6 0-3.2 0 0 1-.3 3.4 1.2a11.5 11.5 0 0 1 6 0c2.3-1.5 3.3-1.2 3.3-1.2.6 1.6.2 2.8 0 3.2.9.8 1.3 1.9 1.3 3.2 0 4.6-2.8 5.6-5.5 5.9.5.4.9 1 .9 2.2v3.3c0 .3.1.7.8.6A12 12 0 0 0 12 .3"
                />
              </svg>
            </a>
          </div>
        </footer>
      </div>
    </div>
  </div>
</div>
`;
